### PR TITLE
feat: add install-steam and enable-flatpak-unfiltered ujust commands

### LINKED ISSUE #639: [FEAT] ujust install-steam
should present the user with the following choices:

1. add the flathub remote and install the steam flatpak, then unset the hardened_malloc var
2. install the bazzite distrobox and export steam
2. install negativo and layer steam

### PR DESCRIPTION
**Pull request by spaceoden**
Resolves #639
All 3 paths have now been tested and are ready to merge.

### CODE REVIEW

File: files/justfiles/utilities.just [hunk 1]
```diff
@@ -146,3 +146,70 @@ toolbox-assemble CONTAINER="prompt" ACTION="create" FILE="/etc/toolbox/toolbox.i
...
+    #!/usr/bin/bash
+    flatpak remote-add --if-not-exists --user flathub https://flathub.org/repo/flathub.flatpakrepo
+
+# Install Steam via choice of 3 methods
+install-steam:
+    #!/bin/bash
+    valid_input="0"
+    method_selection=""
+    echo "Please select a method to install steam:"
+    echo "    0) Cancel - do not install Steam."
```
Comment (RoyalOughtness) [inline files/justfiles/utilities.just]: probably not needed, we don't do this elsewhere

ctrl+c is a universal cancel

File: files/justfiles/utilities.just [hunk 2]
```diff
@@ -146,3 +146,87 @@ toolbox-assemble CONTAINER="prompt" ACTION="create" FILE="/etc/toolbox/toolbox.i
...
+    flatpak remote-add --if-not-exists --user flathub https://flathub.org/repo/flathub.flatpakrepo
+
+# Install Steam via choice of 3 methods
+install-steam:
+    #!/bin/bash
+    if [ -n "$(rpm-ostree status | grep 'kinoite\|sericea\|silverblue\|sway')" ] ; then
+        echo "Steam requires Xwayland, which your variant of secureblue has disabled by default."
+        echo "If you have not already done so, please run 'ujust toggle-xwayland' and reboot to enable it."
+        toggle_xwayland_now=""
+        read -p "Would you like to toggle Xwayland now? [y/N] " toggle_xwayland_now
```
Comment (RoyalOughtness) [inline files/justfiles/utilities.just]: this should detect whether xwayland is already enabled, and skip all of this if it is

File: files/justfiles/utilities.just [hunk 3]
```diff
@@ -146,3 +146,87 @@ toolbox-assemble CONTAINER="prompt" ACTION="create" FILE="/etc/toolbox/toolbox.i
...
+            echo "Distrobox method selected."
+            echo "Creating bazzite-arch distrobox."
+            distrobox-create --unshare-netns --nvidia --image ghcr.io/ublue-os/bazzite-arch --name bazzite-arch -Y
+            echo "Exporting Steam from bazzite-arch distrobox."
+            distrobox-enter -n bazzite-arch -- distrobox-export --app steam
+            ;;
+        3)
+            echo "Native layer method selected."
+            echo "Enabling negativo17 Steam repo."
+            run0 curl -o /etc/yum.repos.d/negativo17-fedora-steam.repo https://negativo17.org/repos/fedora-steam.repo
```
Comment (RoyalOughtness) [inline files/justfiles/utilities.just]: This isn't needed anymore

https://github.com/secureblue/secureblue/pull/663

**Review by RoyalOughtness — APPROVED** [review]

**Review by mkkvcs — APPROVED** [review]

File: files/justfiles/utilities.just [hunk 4]
```diff
@@ -146,3 +146,85 @@ toolbox-assemble CONTAINER="prompt" ACTION="create" FILE="/etc/toolbox/toolbox.i
...
+                systemctl reboot
+            fi
+        fi
+    fi
+    valid_input="0"
+    method_selection=""
+    echo "Please select a method to install steam:"
+    echo "    1) Flatpak - will enable the flathub-unverified repo"
+    echo "    2) Distrobox - will configure the bazzite-arch distrobox image"
+    echo "    3) Native layer - will enable the negativo17 Steam repo"
```
Comment (RoyalOughtness) [inline files/justfiles/utilities.just:179]: just noticed this note is no longer accurate, i'll fix it in a separate PR

**Pull request merged**